mr-runtime/LICENSE

```diff
@@ -0,0 +1,581 @@
+
```

Comment:
I copied this and `NOTICE` from `iceberg-spark-runtime` do I need to modify them in any way? Can we use the files from the project root instead?

Reply:
This is a good start, but we will need to modify them.

LICENSE and NOTICE content is specific to what is actually in the Jar, so we will need to remove the license information for any library that isn't in the Hive runtime and add any new ones that weren't in Spark. It just requires going through the Jar file and making sure all the libraries are accounted for.

Reply:
If you want, we can get the module out with a follow-up item to go over the LICENSE and NOTICE files. I know that's a difficult task for anyone who hasn't done it before. I normally go through the runtime modules just before a release and update them.

Reply:
I'll give this a go, shouldn't be too hard.